.github/actions/container-scanning/action.yml

```diff
@@ -0,0 +1,58 @@
+name: Container Scanning
+description: 'A GitHub Action to scan a container image for security vulnerabilities that follows CZI Best Practices'
+inputs:
+  image-tag:
+    description: 'which image to scan'
+    required: true
+  critical_threshold:
+    description: 'number of critical vulnerabilities that will cause the action to fail'
+    required: false
+    default: "1"
+  high_threshold:
+    description: 'number of high vulnerabilities that will cause the action to fail'
+    required: false
+    default: "1"
+  medium_threshold:
+    description: 'number of medium vulnerabilities that will cause the action to fail'
+    required: false
+    default: "10"
+  low_threshold:
+    description: 'number of low vulnerabilities that will cause the action to fail'
+    required: false
+    default: "10"
+  other_threshold:
+    description: 'number of other vulnerabilities that will cause the action to fail'
+    required: false
+    default: "10"
+  fail_on_vulnerabilities:
+    description: 'whether to fail the action if vulnerabilities are found'
+    required: false
+    default: "true"
+runs:
+  using: "composite"
+  steps:
+    - uses: aws-actions/configure-aws-credentials@v3
+      with:
+        aws-region: us-west-2
+        role-to-assume: arn:aws:iam::871040364337:role/ci/github-actions-inspector
```

Comment:
Is this supposed to be the sec account or should it be core-platform-prod (where the images are pushed)

Reply:
This is independent of accounts. It scans the image that's built in the pipeline. So I am using the sec account